fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -434,7 +434,12 @@ public Response updateSparql(@ContentLocation final InputStream requestBodyStrea
             }
             session.save();
 
-            addCacheControlHeaders(servletResponse, resource(), session);
+            if (resource() instanceof FedoraBinary) {
```

Comment:
Is there a reason this same logic is not also included here:
https://github.com/acoburn/fcrepo4/blob/89f14e961631ba32b2525aeb36f58c648013683d/fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java#L597